taskcluster/scripts/build/macos_build.sh

```diff
@@ -67,34 +67,32 @@ print Y "Updating submodules..."
 git submodule init || die
 git submodule update || die
 
-
-if [[ "$RELEASE" ]]; then
-    # Install dependendy got get-secret.py 
-    python3 -m pip install -r taskcluster/scripts/requirements.txt --user
-    print Y "Fetching tokens..."
-    # Only on a release build we have access to those secrects. 
-    ./taskcluster/scripts/get-secret.py -s project/mozillavpn/tokens -k sentry_dsn -f sentry_dsn
-    ./taskcluster/scripts/get-secret.py -s project/mozillavpn/tokens -k sentry_envelope_endpoint -f sentry_envelope_endpoint
-    export SENTRY_ENVELOPE_ENDPOINT=$(cat sentry_envelope_endpoint)
-    export SENTRY_DSN=$(cat sentry_dsn)
+# Install dependendy got get-secret.py 
+python3 -m pip install -r taskcluster/scripts/requirements.txt
+print Y "Fetching tokens..."
+# Only on a release build we have access to those secrects. 
```

Comment:
So, just out of curiosity, what happens when running these scripts on a PR/Debug build?

Reply:
Ahh that comment is out of date. 
- Given we now have a req that all Taskcluster jobs are coming from people with repo-write access  we now have a  `project/mozillavpn/level-1` namespace for secrets, thereforce PR now can access those tokens 😅